app/routes/_gcn.circulars.$circularId.($version)/route.tsx

```diff
@@ -48,20 +50,24 @@ export async function loader({
     version ? parseFloat(version) : undefined
   )
 
-  return json(result, {
-    headers: {
-      ...publicStaticShortTermCacheControlHeaders,
-      ...getCanonicalUrlHeaders(new URL(`/circulars/${circularId}`, origin)),
-    },
-  })
+  return json(
+    { result },
```

Comment:
Why did you wrap the result in an otherwise empty object?

Reply:
Probably another holdover from something I tried and didn't end up going with. Will also remove